backend/src/test/scala/com/softwaremill/adopttapir/starter/StarterServiceITTest.scala

```diff
@@ -17,11 +17,13 @@ class StarterServiceITTest extends BaseTest with ParallelTestExecution {
     effect <- EffectRequest.values
```

Comment:
What do you think about creating an IT test  where we run: 

"sbt run " in one thread and call /metrics from another? It will then check if metrics are really available under endpoint.

Reply:
I like the idea but see the following issues with that:
* we would need some means to wait until the endpoint is ready and stop it once the verification is done
* it would further extend the tests run (and it will increase exponentially with number of options) - I can imagine it is done as the next (conditional) step of [StarterServiceITTest.scala#L44](https://github.com/softwaremill/adopt-tapir/blob/main/backend/src/test/scala/com/softwaremill/adopttapir/starter/StarterServiceITTest.scala#L44)

Should we add it as a follow up? WDYT?

Reply:
According to documentation: 
https://www.lihaoyi.com/post/HowtoworkwithSubprocessesinScala.html#osprocspawn there is possibility to spawn 2 processess and feed one from other's output. So technically there is no need to introduce wait. Just getting that service is up and running. (We have welcome input for all implementation I guess) 

The another approach will be to add separate test in generated zip project which will run server and check if metrics are available.  Then Integration tests will not be touched. 

It's up to Your preference to split this into separate issue or do it in this task. I am good with both directions.